argo_jupyter_scheduler/executor.py

```diff
@@ -490,6 +554,32 @@ def update_cron_workflow(
         logger.info("cron workflow updated")
 
 
+def main_container(job, use_conda_store_env, input_path, log_path, parameters):
+    envs = []
+    if parameters is not None:
+        for key, value in parameters.items():
+            envs.append(Env(name=key, value=value))
+
+    output_path = gen_default_output_path(input_path)
+    html_path = gen_default_html_path(input_path)
```

Comment:
Note: here I use the `default` versions of these functions. This is because we cannot have access to the `start_time` here. So I use names with the `-unknown` suffix that are renamed in the `rename-files` step (to a file with a proper timestamp that the original jupyter-scheduler would expect).